bin.js

```diff
@@ -31,3 +34,12 @@ function getConfig() {
   }
 }
 
+function getRules(set) {
+  var configFileName = set ? ('eslint-config-' + set) : null
```

Comment:
> You could still use getConfigForFile and then just compare it against local config file, and report missing keys

That would be another way to determine available rules (rather than just reading the folders in the rules directory). But that wouldn't really help us determine whether they're explicitly configuring all rules (including `extends`).

I think our best bet is to probably rewrite the logic to resolve `extends` :-/